src/Sentry/HubExtensions.cs

```diff
@@ -157,5 +157,40 @@ public LockedScope(IHub hub)
 
             public void Dispose() => _scope.Dispose();
         }
+
+        /// <summary>
+        /// Captures the exception with a configurable scope callback.
+        /// </summary>
+        /// <param name="hub">The Sentry hub.</param>
+        /// <param name="ex">The exception.</param>
+        /// <param name="scope">The callback to configure the scope.</param>
+        /// <returns>The Id of the event</returns>
+        public static SentryId CaptureException(this IHub hub, Exception ex, Action<Scope> scope)
```

Comment:
I'd name this `configureScope` instead:

```suggestion
        /// <param name="configureScope">The callback to configure the scope.</param>
        /// <returns>The Id of the event</returns>
        public static SentryId CaptureException(this IHub hub, Exception ex, Action<Scope> configureScope)
```

Reply:
If renamed, to avoid verify errors, don't forget to re-verify or run the tests in a release on your machine before committing.